test/jdk/javax/swing/JFileChooser/FileSizeCheck.java

```diff
@@ -41,35 +41,40 @@
 import java.nio.file.Path;
 import javax.swing.JFileChooser;
 import java.io.RandomAccessFile;
+import javax.swing.WindowConstants;
 
 public class FileSizeCheck {
 
     private static JFrame frame;
+    private static JFileChooser fc;
+    private static PassFailJFrame passFailJFrame;
+    private static Path [] tempFilePaths;
     private static final String INSTRUCTIONS =
             "Click on the \"Details\" button in right-top corner.\n\nScroll Down if required. \n\n" +
-                    "Test 1: If the size of 2.5-KB-File shows 2.5 KB\n" +
-                    "Test 2: If the size of 2.8-MB-File shows 2.8 MB\n" +
-                    "Test 3: If the size of 999-KB-File shows 999 KB\n" +
-                    "Test 4: If the size of 1000-KB-File shows 1 MB\n" +
-                    "Test 5: If the size of 2047-Byte-File shows 2 KB\n" +
-                    "Test 6: If the size of Empty-File shows 0 KB\n\n" +
-                           " press PASS.\n\n";
+                    "Test 1: If the size of 1-Empty-File shows 0 KB\n" +
+                    "Test 2: If the size of 2-File-2047-Byte shows 2.0 KB\n" +
+                    "Test 3: If the size of 3-File-2.5-KB shows 2.5 KB\n" +
+                    "Test 4: If the size of 4-File-999-KB shows 999.0 KB\n" +
+                    "Test 5: If the size of 5-File-1000-KB shows 1.0 MB\n" +
+                    "Test 6: If the size of 6-File-2.8-MB shows 2.8 MB\n\n" +
+                           "press PASS.\n\n";
 
-    public static void test() {
-        JFileChooser fc = new JFileChooser();
+    public static void test() { //throws InterruptedException, InvocationTargetException {
+        frame = new JFrame("JFileChooser File Size test");
+        fc = new JFileChooser();
         Path dir = Paths.get(System.getProperty("test.src"));
-        String [] tempFilesName = {"2.5-KB-File","2.8-MB-File","999-KB-File","1000-KB-File","2047-Byte-File","Empty-File"};
-        int [] tempFilesSize = {2500, 2800000,999000,1000000,2047,0};
-        Path [] tempFilePaths = new Path[tempFilesName.length];
-        for (int i = 0 ; i < tempFilesName.length ; i++) {
-            tempFilePaths[i] = dir.resolve(tempFilesName[i]);
-        }
-
+        String [] tempFilesName = {"1-Empty-File", "2-File-2047-Byte", "3-File-2.5-KB", "4-File-999-KB", "5-File-1000-KB", "6-File-2.8-MB"};
```

Comment:
Please break the long line. It's a common practice to code into 80 columns, or 100 at maximum. This line has 140 columns.